console/web3ext/web3ext.go

```diff
@@ -1487,6 +1487,14 @@ web3._extend({
 			name: 'childOperatorFeePayer',
 			getter: 'subbridge_getChildOperatorFeePayer',
 		}),
+		new web3._extend.Property({
+			name: 'ParentBridgeOperatorGasLimit',
```

Comment:
Could you follow the naming convention? All the other APIs start with lower case.

Reply:
I'll update the code. Thanks for the review.